httplib2shim/__init__.py

```diff
@@ -42,6 +44,8 @@ def _default_make_pool(http, proxy_info):
     cert_reqs = 'CERT_REQUIRED' if http.ca_certs and not ssl_disabled else None
 
     if proxy_info:
+        proxy_info = proxy_info()
```

Comment:
Can you remind me what `proxy_info()` does? Does it use no proxy at all? Reading this code it seems that this code will now always return a `ProxyManager` - is that what we want when `proxy_info` is None?